plugins/postcss-global-data/test/examples/example.css

```diff
@@ -0,0 +1,7 @@
+.foo {
```

Comment:
I don't think the examples are used?
Can we delete these?

Reply:
Yep will do!